app/controllers/comments_controller.rb

```diff
@@ -23,4 +23,14 @@ def create
       end
     end
   end
+
+  def destroy
+    comment = Comment.find(params[:id])
```

Comment:
After a comment is deleted, I noticed that the `comments_counter` is not updated on the page. The post still indicated that there was a comment even when there were no comments on the post.

- Kindly look into it and make sure that everything is consistent on the application including the counters.